x-pack/plugins/security_solution/server/fleet_integration/handlers/validate_endpoint_package_policy.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import moment from 'moment';
+
+import type { NewPackagePolicyInput } from '@kbn/fleet-plugin/common';
+
+export const validateEndpointPackagePolicy = (input: NewPackagePolicyInput) => {
+  if (input.config?.policy?.value?.global_manifest_version) {
+    const globalManifestVersion = input.config.policy.value.global_manifest_version;
+
+    if (globalManifestVersion !== 'latest') {
+      const parsedDate = moment(globalManifestVersion, 'YYYY-MM-DD', true);
+      if (!parsedDate.isValid()) {
+        throw createManifestVersionError(
+          'Invalid date format. Use "latest" or "YYYY-MM-DD" format.'
+        );
+      }
+
+      const maxAllowedDate = moment().subtract(18, 'months');
+      if (parsedDate.isBefore(maxAllowedDate)) {
+        throw createManifestVersionError(
+          'Global manifest version is too far in the past. Use "latest" or a date within the last 18 months.'
+        );
+      }
+      if (parsedDate.isAfter(moment())) {
```

Comment:
I'm not deeply familiar with `moment`, but a quick search suggested that it creates date objects in the local timezone.

can it happen that a user selecting today's date in their browser's timezone let's say at 2am, receives an error when trying to save the config, because it's still one day earlier in the timezone where Kibana server is?

Reply:
Yup, great point. Replaced all `moment()` with `moment.utc()` just to be on the safe side that both EuICalendar and our backend uses it. Thanks!

Reply:
Thats safer, but since this is backend code, we should be good right? We should ensure it is utc at UI side I guess

Reply:
I changed it UI side as well.